text/0002-quality-check-workflow.md

```diff
@@ -0,0 +1,225 @@
+| Transcript Coverage and 5'-3' Bias | [Qualimap]  | Libraries prepared with polyA selection may have higher biased expression in 3' region. If reads primarily accumulate at the 3' end of transcripts (in poly(A)-selected samples), this might indicate the starting RNA was of low quality.                                                                                                                                                                |
+| Junction Analysis                  | [Qualimap]  | Analysis of known, partly known, and novel junction positions in spliced alignments.                                                                                                                                                                                                                                                                                                                      |
+| Strand Specificity                 | ngsderive   | Verification/sanity check of how reads were stranded for the RNA sequencing (stranded or unstranded protocol).                                                                                                                                                                                                                                                                                            |
+| GC Content Bias                    | ?           | GC profiles are typically remarkably stable. Even small/minor deviations could indicate a problem with the library used (or bacterial contamination).                                                                                                                                                                                                                                                     |
+
+#### Thresholds and Metrics for Specific Applications
+
+To apply quality control metrics to vett data, we need reasonable thresholds that are practically acheivable and neither too lax or too strict. Our preference is for statistically or empirically determined thresholds rather than arbitrary estimates. By statistical thresholds, we are referring to distributional tests that formally define outliers. By empirical thresholds, we are referring to standards below which data analysis or interpretation are degraded. Statistical tests can be performed on large populations of QC data. We are already in postion to do that today. Empirical tests, however, require foreknowledge of the correct results. This requires experimental design and implementation through a laboratory at some cost.
```

Comment:
```suggestion
To apply quality control metrics to vet data, we need reasonable thresholds that are practically achievable and neither too lax nor too strict. Our preference is for statistically or empirically determined thresholds rather than arbitrary estimates. By statistical thresholds, we are referring to distributional tests that formally define outliers. By empirical thresholds, we are referring to standards below which data analysis or interpretation are degraded. Statistical tests can be performed on large populations of QC data. We are already in a position to do that today. Empirical tests, however, require foreknowledge of the correct results. This requires experimental design and implementation through a laboratory at some cost.
```